lib/backpex/resource_action.ex

```diff
@@ -54,11 +54,11 @@ defmodule Backpex.ResourceAction do
             ) :: Ecto.Changeset.t()
 
   @doc """
-  The handle function for the corresponding action. It receives the params and will be called when the form is valid and submitted.
+  The handle function for the corresponding action. It receives the socket and casted and validated data (received from [`Ecto.Changeset.apply_changes/2`](https://hexdocs.pm/ecto/Ecto.Changeset.html#apply_action/2)) and will be called when the form is valid and submitted.
```

Comment:
```suggestion
  The handle function for the corresponding action. It receives the socket and casted and validated data (received from [`Ecto.Changeset.apply_changes/1`](https://hexdocs.pm/ecto/Ecto.Changeset.html#apply_action/2)) and will be called when the form is valid and submitted.
```

https://hexdocs.pm/ecto/Ecto.Changeset.html#apply_changes/1